Source/Templates/GtkSharp.Template.CSharp/content/GtkSharp.Application.CSharp/.template.config/template.json

```diff
@@ -18,8 +18,27 @@
   "symbols": {
     "targetframework": {
       "type": "parameter",
-      "defaultValue": "net5.0",
-      "replaces": "net5.0"
+      "description": "The target framework for the project.",
+      "datatype": "choice",
```

Comment:
Hmm, will this lock us into selecting only one of the three possible frameworks, or can we still specify any target framework during creation and this just helps with the documentation for the command?

Reply:
I believe this is list of all possible values. Do you need additional frameworks? I list all except .Net Framework family here, can add that too.

Reply:
I would like to leave it as it was, as when .net 7 releases, you will be able to create the project with that as the target framework, without any update to the templates.